### ISSUE TITLE
How to set learning rate

### ISSUE THREAD

**Issue by universewill** [junyanz/pytorch-CycleGAN-and-pix2pix#559]
I want to set learning rate for train, but i do not know how to set it

**Comment by abc99lr** [comment]
The model schedule is defined at https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/networks.py#L31
